app/main.py

```diff
@@ -1,34 +1,63 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self,
+                 row: int,
+                 column: int,
+                 is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self,
+                 start: tuple,
+                 end: tuple,
+                 is_drowned: bool = False) -> None:
+        self.start = start
+        self.end = end
+        self.is_drowned = is_drowned
+        self.decks = []
+        if self.start[0] == self.end[0] and self.start[1] == self.end[1]:
+            self.decks = [Deck(self.start[0], self.start[1])]
+            return
```

Comment:
I don't like this empty return, but it is not a mistake

Reply:
you can use `elif` in the next lines, so here you will not need this `return`. WDYT?